### PR TITLE
Reminder Model

### PR DESCRIPTION
**Pull request by martyschaer**

### CODE REVIEW

**Comment by martyschaer** [comment]
@haesler could you please also review the parts relevant for you?
(I think [Recurrence](https://github.com/ch-bfh-bti7081-s2019-green/ch.bfh.bti7081.s2019.green/blob/d8f454ec71e6f6d04e3ffcecd64b63cef2739d29/src/main/java/ch/bfh/bti7081/s2019/green/model/reminder/Recurrence.java) mostly)

File: src/test/java/ch/bfh/bti7081/s2019/green/persistence/dao/PersonDaoTest.java [hunk 1]
```diff
@@ -25,15 +25,15 @@
 
     @BeforeClass
     public static void setup(){
-        Person james = savePerson(1337L,
+        Person james = savePerson(1337,
```
Comment (severinkaderli) [inline src/test/java/ch/bfh/bti7081/s2019/green/persistence/dao/PersonDaoTest.java:28]: Is the id here even necessary? Doesn't AbstractEntity set an incremented key automatically?

Comment (martyschaer) [reply src/test/java/ch/bfh/bti7081/s2019/green/persistence/dao/PersonDaoTest.java:28]: This isn't an ID, but the AHV number. I renamed the param to be more clear about that.

Comment (severinkaderli) [reply src/test/java/ch/bfh/bti7081/s2019/green/persistence/dao/PersonDaoTest.java:28]: > This isn't an ID, but the AHV number. I renamed the param to be more clear about that.

Ah, in that case I'm ok with the PR. :+1:

**Pull request merged**